feat(server): add new blog post about website redesign (#8571)
tuist/tuist: server/priv/marketing/blog/2025/11/03/website-redesign.md

![Home page of the new marketing site](/marketing/images/blog/2025/11/03/website-redesign/home-page.png)

## The process

Before redesigning the website, we spent time researching what could be improved and what was missing. We reviewed over 25 developer tool websites, studying their strengths and weaknesses. The website plays a big role in shaping our brand identity, and with the dashboard redesign, we wanted to bring the same look and feel to our site.

![Website moodboard](/marketing/images/blog/2025/11/03/website-redesign/moodboard.png)

The main issues we identified were:
* The old website didn’t clearly communicate that Tuist had grown beyond the CLI.
* The previous site layout made it hard to find information or understand Tuist’s features.
* It was difficult to showcase new features and updates effectively.
* We weren’t clearly demonstrating how Tuist supports real teams or inspire new contributors to get involved.
* The website visuals didn’t align with the newly redesigned dashboard.

### Stage 1: Clarifying the Message

> The first thing we noticed was that our story wasn’t being told the way we wanted. Tuist had grown far beyond being just a CLI tool, yet our website still gave that impression. We wanted to communicate what Tuist truly is today, a platform that helps teams build, collaborate, and ship iOS apps faster.

Reviewer: ```suggestion
The first thing we noticed was that our story wasn’t being told the way we wanted. Tuist had grown far beyond being just a CLI tool, yet our website still gave that impression. We wanted to communicate what Tuist truly is today, a platform that helps teams build, collaborate, and ship mobile apps.
```

> The first thing we noticed was that our story wasn’t being told the way we wanted it to be. Tuist had grown far beyond being just a CLI tool, yet our website still gave that impression. We wanted to communicate what Tuist truly is today, a platform that helps teams build, collaborate, and ship iOS apps faster.

Reviewer: ```suggestion
The first thing we noticed was that our story wasn’t being told the way we wanted it to be. Tuist had grown far beyond being just a CLI tool, yet our website still gave that impression. We wanted to communicate what Tuist truly is today, a platform that helps teams build, collaborate, and ship apps.
```

I'm somewhat allergic to the "... apps faster." I'd personally not use that messaging anymore

To do this, we reworked the content and page flow to focus on the bigger picture of Tuist’s ecosystem. We looked at developer tools like [Vercel](https://vercel.com/home), [Supabase](https://supabase.com/), and [Linear](https://linear.app/homepage), understanding how they convey complex ideas in a clear, simple way. More on this section below.

![Website header exploration](/marketing/images/blog/2025/11/03/website-redesign/headers.png)

### Stage 2: Improving the Structure and Navigation

> Another issue we saw was how difficult it was to find information or understand what Tuist offered at a glance. The navigation felt fragmented and didn’t guide new visitors through the story.

Reviewer: ```suggestion
Another issue we saw was how difficult it was to understand what Tuist offered at a glance. The navigation felt fragmented and didn’t guide new visitors through the story.
```

what information? Without further specifying, I'd just cut that piece

![Tuist feature bento cards](/marketing/images/blog/2025/11/03/website-redesign/bento-cards.png)

### Stage 4: Showcasing Customer Stories and Community Impact

Being an open-source company, we’ve always valued transparency and community involvement. But the old site didn’t really show how Tuist was helping teams or how contributors were shaping the project. 

We added customer case studies with custom illustrations to tell real stories, how teams reduced build times, improved collaboration, and scaled their development workflows using Tuist. 

We also introduced community metrics to highlight our open-source growth and inspire more developers to contribute.

![Customer case study cards](/marketing/images/blog/2025/11/03/website-redesign/case-studies.png)

![Stats for Tuist project](/marketing/images/blog/2025/11/03/website-redesign/stats.png)

## Unifying the Brand and Visual Language

> Finally, we wanted to make sure our website felt like a natural extension of our newly redesigned dashboard. The previous site had a different tone and visual style, which made the overall experience feel disconnected. We brought everything together under a unified brand identity and built the website using the Noora Design System. All core elements, buttons, input fields, shadows, colors, and variables, are based on Noora for consistency and scalability.

Reviewer 2: ```suggestion
Finally, we wanted to make sure our website felt like a natural extension of our newly redesigned dashboard. The previous site had a different tone and visual style, which made the overall experience feel disconnected. We brought everything together under a unified brand identity and built the website using the [Noora Design System](https://github.com/tuist/noora). All core elements, buttons, input fields, shadows, colors, and variables, are based on Noora for consistency and scalability.
```
